src/system/UsersModule/Helper/MailHelper.php

```diff
@@ -101,11 +110,14 @@ public function createAndSendRegistrationMail(UserEntity $userEntity, $userNotif
         }
         if ($adminNotification) {
             // mail notify email to inform admin about registration
-            $mailEmail = $this->variableApi->get('ZikulaUsersModule', UsersConstant::MODVAR_REGISTRATION_ADMIN_NOTIFICATION_EMAIL, '');
-            if (!empty($mailEmail)) {
-                $mailSent = $this->sendNotification($mailEmail, 'regadminnotify', $rendererArgs);
+            $notificationEmail = $this->variableApi->get('ZikulaUsersModule', UsersConstant::MODVAR_REGISTRATION_ADMIN_NOTIFICATION_EMAIL, '');
+            if (!empty($notificationEmail)) {
+                $authMapping = $this->authenticationMappingRepository->getByZikulaId($userEntity->getUid());
```

Comment:
not sure about this. Every user will not have an entry in this table. for example, if a user registers with the OAuth module, they will not have an entry. The UsersModule will have a uname and email stored for each user regardless of registration/login method.